setup.py

```diff
@@ -2,6 +2,7 @@
 
 install_requires = [
     'prometheus-client==0.11.0',
+    'celery'
```

Comment:
since it's only been tested unser celery, `4.X.X` - Mention constraint here as well

Reply:
locked version to 5.2.7 since this version is being used in both IB and westeros